### PR TITLE
added 8.2 flask and feast to monk/shaman/hunter/death knight

### PR DESCRIPTION
**Pull request by MayContainNutz**
battle potions remain untouched

### CODE REVIEW

File: engine/class_modules/sc_monk.cpp [hunk 1]
```diff
@@ -9366,7 +9366,7 @@ std::string monk_t::default_flask() const
       break;
     case MONK_MISTWEAVER:
       if ( true_level > 110 )
-        return "endless_fathoms";
+        return "greater_flask_of_endless_fathoms";
```
Comment (kendrick-walls-work) [inline engine/class_modules/sc_monk.cpp:9369]: ⚠️ This conflicts with PR simulationcraft/simc#4942 commit 8e29110851d501980edaf96dcf988a3d52522965

However it is possibly a more appropriate fix. Either way the merge conflict resolution should be trivial.

**Comment by kendrick-walls-work** [comment]
I'm working on a conflict resolution merge

**Comment by kendrick-walls-work** [comment]
:x: these changes have been assimilated into PR #4948
RECOMMEND CLOSING THIS OBSOLETE PULL REQUEST
rational: keep this PR for the comment history if ever needed but close as resolved to avoid detached state in the future

**Comment by kendrick-walls-work** [comment]
❌ these changes have been cherry picked into PRs #4949 #4950
RECOMMEND CLOSING THIS OBSOLETE PULL REQUEST
rational: keep this PR for the comment history if ever needed but close as resolved to avoid detached state (merge conflicts) and confusion and cross team dependancies in the future

**Comment by Melekus** [comment]
Most of the proposed changes have been applied individually in other commits, closing the PR.

**Pull request closed without merging**